Userland/Libraries/LibGfx/CMakeLists.txt

```diff
@@ -78,3 +77,26 @@ set(SOURCES
 
 serenity_lib(LibGfx gfx)
 target_link_libraries(LibGfx PRIVATE LibCompress LibCore LibCrypto LibFileSystem LibTextCodec LibIPC LibUnicode)
+
+set(generated_sources TIFFMetadata.h TIFFTagHandler.cpp)
+list(TRANSFORM generated_sources PREPEND "ImageFormats/")
+
+find_package(Python COMPONENTS Interpreter REQUIRED)
+
+add_custom_command(
+        OUTPUT  ${generated_sources}
+        COMMAND "${CMAKE_COMMAND}" -E make_directory "tmp"
+        COMMAND ${Python_EXECUTABLE} "${CMAKE_CURRENT_SOURCE_DIR}/tiff_generator.py" -o "${CMAKE_CURRENT_BINARY_DIR}/tmp"
+        COMMAND "${CMAKE_COMMAND}" -E copy_if_different tmp/TIFFMetadata.h "ImageFormats/TIFFMetadata.h"
+        COMMAND "${CMAKE_COMMAND}" -E copy_if_different tmp/TIFFTagHandler.cpp "ImageFormats/TIFFTagHandler.cpp"
+        COMMAND "${CMAKE_COMMAND}" -E remove_directory "${CMAKE_CURRENT_BINARY_DIR}/tmp"
+        VERBATIM
```

Comment:
can we update the python script do to the "write if different" step itself? It should :tm: be slightly faster than invoking an extra 4 cmake commands. I did this for IDLGenerator and it worked decently.